switss/model/dtmc.py

```diff
@@ -66,19 +73,10 @@ def standard_trans_map(sourceidx, destidx, p):
         :rtype: graphviz.Digraph
         """ 
 
-        def standard_state_map(stateidx,labels):
-            return { "color" : color_from_hash(tuple(sorted(labels))),
-                     "label" : "State %d\n%s" % (stateidx,",".join(labels)),
-                     "style" : "filled" }
-
-        def standard_trans_map(sourceidx, destidx, p):
-            return { "color" : "black", 
-                     "label" : str(round(p,10)) }
-
-        state_map = standard_state_map if state_map is None else state_map
-        trans_map = standard_trans_map if trans_map is None else trans_map
+        state_map = self.visualization.state_map if state_map is None else state_map
+        trans_map = self.visualization.trans_map if trans_map is None else trans_map
 
-        dg = Digraph()
+        dg = Digraph(**kwargs)
```

Comment:
This is problematic, since the key "action_map=x" is sometimes added to calls of .digraph. If the underlying model is a DTMC, this will add this as a key/value-pair to kwargs, since there is no formal parameter "action_map" in the method head. But then dg = Digraph(action_map="..") will yield an exception or something similar. 

One could add an "action_map" parameter to DTMC's method head, but this makes no sense, since there are no actions. I think we should remove this.